src/devices/esp8266.c

```diff
@@ -470,11 +468,13 @@ bool esp8266_join_ap(const char* ssid, const char* pass, void (*cb)(bool))
                 return false;
 
         char cmd[64];
-        if (pass && *pass)
-                snprintf(cmd, ARRAY_LEN(cmd), "AT+CWJAP=\"%s\",\"%s\"",
-                         ssid, pass);
-        else
-                snprintf(cmd, ARRAY_LEN(cmd), "AT+CWJAP=\"%s\",", ssid);
+        const char pfx[] = "AT+CWJAP_DEF";
```

Comment:
we can do _CUR or _DEF - I agree with using _DEF so it comes up that way upon power up. :+1: